source/releasenotes/compat.rst

```diff
@@ -22,8 +22,8 @@ Supported OS Versions for Management Server
 This section lists the operating systems that are supported for running
 CloudStack Management Server.
 
--  Ubuntu 16.04 LTS, 18.04, 20.04 LTS
--  CentOS versions 7.x, 8.x
+-  Ubuntu 16.04 LTS, 18.04 LTS, 20.04 LTS
```

Comment:
Ubuntu 16.04 also has an EOL coming soon, in April 2021. Should add such a disclaimer for it as well?

Reply:
@GabrielBrascher it's a normal case where the EOL date hasn't changed and I suppose most users would be aware that 16.04 is old enough by now.